src/components/Booking/RequestButton/index.js

```diff
@@ -33,8 +34,11 @@ const RequestButton = (props) => {
       />
     ) : (
       <Button
-        disabled={(remainingHolidays <= 0
-          || availableDays < 0)}
+        disabled={eventType !== eventTypes.WFH
+          ? ((remainingHolidays <= 0
+            || availableDays < 0))
```

Comment:
can you pull this to the line above, it looks bad, but I bet esLint suggested it haha

also should only need one set of brackets

`? (remainingHolidays <= 0 || availableDays < 0)`

Reply:
haha i have put it onto one line